.github/workflows/ci.yml

```diff
@@ -17,6 +17,9 @@ on:
       - setup.py
       - versioneer.py
   push:
+  schedule:
```

Comment:
Should this run only on push to main?

Reply:
I don't think so. The load on CI resources will be greater. If necessary, in the future we can simply increase the launch frequency (not once a week, but twice, for example).